### PR TITLE
Add a Dockerfile

### PR DESCRIPTION
**Pull request by steved**
@zendesk/runway

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ADD plugins /app/plugins
+
+RUN bundle install --without test sqlite postgres --quiet --local --jobs 4 || bundle check
+
+# Code
+ADD config /app/config
+ADD app /app/app
+ADD lib /app/lib
+
+RUN DATABASE_URL=mysql2://user:pass@127.0.0.1/null RAILS_ENV=development PRECOMPILE=1 bundle exec rake --trace assets:precompile
```
Comment (sandlerr) [inline Dockerfile]: why this db config? Rails 4 and up always tries to connect even if you're precompiling?

Comment (steved555) [inline Dockerfile]: Yeah, it looks for a configuration, even if it doesn't connect.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
+FROM jacobat/ruby:2.2.1
```
Comment (grosser) [inline Dockerfile]: hmm this is unusable for anyone outside of zendesk ... some open-source image would be nice ...

File: config/puma.rb [hunk 3]
```diff
@@ -1,4 +0,0 @@
-threads 8,250
```
Comment (grosser) [inline config/puma.rb]: where did that 8,250 config go ?

**Comment by grosser** [comment]
:+1:

File: Dockerfile (hunk 2 above)
Comment (steved555) [inline Dockerfile]: Right, thanks.

File: config/puma.rb (hunk 3 above)
Comment (steved555) [inline config/puma.rb]: This file wasn't loaded by `rails server`, so it seems kind of pointless. Unless people actually run `puma` directly.

**Comment by sandlerr** [comment]
Already merged

**Pull request closed without merging**